addons/mail/models/ir_attachment.py

```diff
@@ -83,6 +85,7 @@ def _to_store(self, store: Store, /, *, fields=None, extra_fields=None):
                 "checksum",
                 "create_date",
                 "filename",
+                "message",
```

Comment:
I don't know if we want to generically return the message for all the attachments all the time as a default value.

Can't we be more specific with extra_fields, for example only attachments fetched through whatever flows leads to displaying the panel?